ipaplatform/debian/tasks.py

```diff
@@ -11,14 +11,20 @@
 from ipaplatform.base.tasks import BaseTaskNamespace
 from ipaplatform.redhat.tasks import RedHatTaskNamespace
 
+from ipapython import ipautil
 
 class DebianTaskNamespace(RedHatTaskNamespace):
     @staticmethod
     def restore_pre_ipa_client_configuration(fstore, statestore,
                                              was_sssd_installed,
                                              was_sssd_configured):
-        # Debian doesn't use authconfig, nothing to restore
-        return True
+        ret = True
+        try:
+            ipautil.run(["pam-auth-update",
+                         "--package", "--remove", "mkhomedir"])
+        except ipautil.CalledProcessError:
+            ret = False
+        return ret
```

Comment:
A little nit:
```python
try:
    ...
except ipautil.CalledProcessError:
    return False
return True
```
The above looks nicer to me :)

Reply:
sure